scripts/travis-e2e.sh

```diff
@@ -36,3 +36,4 @@ kubectl apply -f scripts/minikube-rbac.yaml
 
 make crossbuild
 make e2e
+make e2e-helm
```

Comment:
I think I'd prefer to have this in a separate job as this gives the notion, that the previous steps are related/required, when they're not